CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+## 0.7.0 - 9 July 2019
+### Added
+* Revamped Docker Explorer
+  * Containers, images, and registries now have their own explorer which can be hidden, resized, or reordered
+  * Added per-explorer settings for display format, grouping, and sorting
+  * Modified icons to respect theme
+  * Moved connection errors and troubleshooting links directly into the explorer instead of as a separate notification
+  * Added support for "Load more..." if not all items are retrieved in the first batch
+  * Local explorers poll less often (only if the explorer is open)
+  * Added per-explorer prune command (system prune is still available from the command palette)
+  * Ensured all desctructive actions have a confirmation and are grouped separately in context menus
+* Generalized registries view to better support more providers
+  * All registries regardless of provider now support viewing repos/tags, pulling images, and setting a registry as default
+  * Added docs for contributing a new registry provider
+  * Multiple registry providers of the same type can now be connected (e.g. multiple Docker Hub accounts)
+  * Added support for GitLab (not including self-hosted)
+* Update to version 0.0.21 of the language server (thanks @rcjsuen)
+  * Improves linting checks so that there are fewer false positives
```

Comment:
ping @rcjsuen in case you want to suggest changes to the language server bullet points

Reply:
Thanks for the heads up, @EricJizbaMSFT. What you’ve written there seems fine to me.